tests/virt_autotest/esxi_open_vm_tools.pm

```diff
@@ -137,16 +164,34 @@ sub get_epoch_time {
 sub init_guest_time {
     my $h_datetime = get_host_timestamp();
     # Get last day of the host time
-    my $last_day = script_output("date -u -d '$h_datetime last day' +'\%F \%T'");
+    my $last_day;
+    if (check_var('BACKEND', 'svirt')) {
+        $last_day = script_output("date -u -d '$h_datetime last day' +'\%F \%T'");
+    }
+    elsif (check_var('BACKEND', 'qemu')) {
+        $last_day = script_output(qq($ssh_vm "date -u -d '$h_datetime last day' +'\%F \%T'"));
+    }
     # Set the guest time by using the variable $last_day
-    my $g_datetime = script_output("date -u -s '$last_day' +'\%F \%T'");
+    my $g_datetime;
+    if (check_var('BACKEND', 'svirt')) {
+        $g_datetime = script_output("date -u -d '$h_datetime last day' +'\%F \%T'");
+    }
+    elsif (check_var('BACKEND', 'qemu')) {
+        $g_datetime = script_output(qq($ssh_vm "date -u -s '$last_day' +'\%F \%T'"));
```

Comment:
Same here? Same cmd is ok?

Reply:
Refer to https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16179#discussion_r1116829329